features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -247,7 +249,8 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
             if (
                 isFeatureEnabled(featureConfig?.applications,
                     ApplicationManagementConstants.FEATURE_DICTIONARY.get("APPLICATION_EDIT_GENERAL_SETTINGS"))
-                && !isSubOrganization()
+                && (isSubOrganization() ?
+                    !brandingDisabledFeatures.includes("branding.applicationLevelBranding"): true)
```

Comment:
Define `branding.applicationLevelBranding` as a constant and use

Reply:
Check in other places as well

Reply:
FIxed https://github.com/wso2/identity-apps/pull/6409/commits/1f0ef2552c269b4a70a36b8a12bb542f60fc1e78